torch/cuda/__init__.py

```diff
@@ -1693,6 +1695,237 @@ def addmm_kernel_impl(*args, **kwargs):
 _lazy_call(_register_triton_kernels)
 
 
+
+
+def compile_kernel(
+    kernel_source: str,
+    kernel_name: str,
+    compute_capability: str = None,
+    header_code: str = "",
+    cuda_include_dirs: list = None,
+    nvcc_options: list = None
+):
+    """
+    Compiles a CUDA kernel using NVRTC and returns a callable function.
+    
+    This function is a wrapper for NVRTC that enables runtime compilation of CUDA kernels
+    to be used with PyTorch tensors. It handles the compilation process and creates a Python
+    callable that can be used to invoke the kernel with PyTorch tensors as arguments.
```

Comment:
Could you add: the generated function is a "raw cuda kernel", to get it to work as a PyTorch op, please follow the guide at https://pytorch.org/tutorials/advanced/python_custom_ops.html to wrap it into a custom op.